### PR TITLE
Update TiFlash scale-in guidance.

### PR DESCRIPTION
**Pull request by JinheLin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by jinwhu — COMMENTED** [review]
查询是否存在 TiFlash 副本数大于缩容后的 TiFlash 节点数的数据表

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
 
 ### 1. 根据 TiFlash 剩余节点数调整数据表的副本数
 
-在下线节点之前，确保 TiFlash 集群剩余节点数大于等于所有数据表的最大副本数，否则需要修改相关表的 TiFlash 副本数。
+1. 查询 TiFlash 副本数大于缩容后的 TiFlash 节点数的数据表。`tobe_left_nodes` 表示缩容后的 TiFlash 节点数。如果查询结果为空，可以开始执行缩容。如果查询结果不为空，则需要修改相关表的 TiFlash 副本数。
 
-1. 在 TiDB 客户端中针对所有副本数大于集群剩余 TiFlash 节点数的表执行：
+	{{< copyable "sql" >}}
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:364]:
```suggestion
```

File: scale-tidb-using-tiup.md [hunk 2]
```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
...
+    ```
+
+2. 对所有 TiFlash 副本数大于缩容后的 TiFlash 节点数的表执行：
 
     {{< copyable "sql" >}}
 
     ```sql
-    alter table <db-name>.<table-name> set tiflash replica 0;
+    alter table <db-name>.<table-name> set tiflash replica ‘tobe_left_nodes’;
     ```
```
Comment (jinwhu) [inline scale-tidb-using-tiup.md:376]: alter table <db-name>.<table-name> set tiflash replica ‘a new number of copies which should be less than obe_left_nodes ’;

File: scale-tidb-using-tiup.md [hunk 3]
```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
...
+    select * from information_schema.tiflash_replica where REPLICA_COUNT >  'tobe_left_nodes'
+    ```
+
+2. 对所有 TiFlash 副本数大于缩容后的 TiFlash 节点数的表执行：
 
     {{< copyable "sql" >}}
 
     ```sql
-    alter table <db-name>.<table-name> set tiflash replica 0;
+    alter table <db-name>.<table-name> set tiflash replica ‘tobe_left_nodes’;
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:375]:
```suggestion
    ALTER TABLE <db-name>.<table-name> SET tiflash replica 'tobe_left_nodes';
```

File: scale-tidb-using-tiup.md [hunk 4]
```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
 
 ### 1. 根据 TiFlash 剩余节点数调整数据表的副本数
 
-在下线节点之前，确保 TiFlash 集群剩余节点数大于等于所有数据表的最大副本数，否则需要修改相关表的 TiFlash 副本数。
+1. 查询 TiFlash 副本数大于缩容后的 TiFlash 节点数的数据表。`tobe_left_nodes` 表示缩容后的 TiFlash 节点数。如果查询结果为空，可以开始执行缩容。如果查询结果不为空，则需要修改相关表的 TiFlash 副本数。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:362]:
```suggestion
1. 查询是否有数据表的 TiFlash 副本数大于缩容后的 TiFlash 节点数。`tobe_left_nodes` 表示缩容后的 TiFlash 节点数。如果查询结果为空，可以开始执行缩容。如果查询结果不为空，则需要修改相关表的 TiFlash 副本数。
```

File: scale-tidb-using-tiup.md [hunk 5]
```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
...
 
-1. 在 TiDB 客户端中针对所有副本数大于集群剩余 TiFlash 节点数的表执行：
+	{{< copyable "sql" >}}
+
+    ```sql
+    select * from information_schema.tiflash_replica where REPLICA_COUNT >  'tobe_left_nodes'
+    ```
+
+2. 对所有 TiFlash 副本数大于缩容后的 TiFlash 节点数的表执行：
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:370]:
```suggestion
2. 对所有 TiFlash 副本数大于缩容后的 TiFlash 节点数的表执行以下语句：
```

File: scale-tidb-using-tiup.md [hunk 6]
```diff
@@ -359,17 +359,23 @@ tiup cluster display <cluster-name>
...
     {{< copyable "sql" >}}
 
     ```sql
-    alter table <db-name>.<table-name> set tiflash replica 0;
+    ALTER TABLE <db-name>.<table-name> SET tiflash replica 'tobe_left_nodes';
     ```
 
-2. 等待相关表的 TiFlash 副本被删除（按照[查看表同步进度](/tiflash/create-tiflash-replicas.md#查看表同步进度)一节操作，查不到相关表的同步信息时即为副本被删除）。
+3. 重新执行步骤 1，确保没有数据表的 TiFlash 副本数大于缩容后的 TiFlash 节点数。
+
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:378]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: scale-tidb-using-tiup.md [hunk 7]
```diff
@@ -359,17 +359,22 @@ tiup cluster display <cluster-name>
 
 ### 1. 根据 TiFlash 剩余节点数调整数据表的副本数
 
-在下线节点之前，确保 TiFlash 集群剩余节点数大于等于所有数据表的最大副本数，否则需要修改相关表的 TiFlash 副本数。
+1. 查询是否有数据表的 TiFlash 副本数大于缩容后的 TiFlash 节点数。`tobe_left_nodes` 表示缩容后的 TiFlash 节点数。如果查询结果为空，可以开始执行缩容。如果查询结果不为空，则需要修改相关表的 TiFlash 副本数。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:363]:
```suggestion
```

**Review by jinwhu — APPROVED** [review]
good

**Pull request merged**